client/python/gradio_client/utils.py

```diff
@@ -27,6 +27,15 @@
 RESET_URL = "/reset"
 DUPLICATE_URL = "https://huggingface.co/spaces/{}?duplicate=true"
 STATE_COMPONENT = "state"
+INVALID_RUNTIME = [
+    "NO_APP_FILE",
+    "CONFIG_ERROR",
+    "BUILDING",
+    "BUILD_ERROR",
+    "RUNTIME_ERROR",
+    "PAUSED",
+    "SLEEPING",
```

Comment:
Should SLEEPING be considered an invalid state? I've used the Client to connect with Spaces that were sleeping and after a few seconds, they wake up and are able to make predictions. I don't think we should exclude such Spaces

Reply:
I assumed there'd be some lag such that the first prediction would fail. Can remove!

Reply:
It took like ~10 seconds IIRC but it went through :)